ui/app/AppLayouts/Chat/CommunityComponents/ImportCommunityPopup.qml

```diff
@@ -0,0 +1,71 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.3
+import QtGraphicalEffects 1.13
+import QtQuick.Dialogs 1.3
+import "../../../../imports"
+import "../../../../shared"
+import "../../../../shared/status"
+
+ModalPopup {
+    property string keyValidationError: ""
+
+    id: popup
+    height: 300
+
+    onOpened: {
+        keyInput.text = isEdit ? community.name : "";
+        keyInput.forceActiveFocus(Qt.MouseFocusReason)
+    }
+
+    function validate() {
+        keyValidationError = ""
+
+        if (keyInput.text === "") {
+            keyValidationError = qsTr("You need to enter a key")
+        }
+
+        return !keyValidationError
+    }
+
+    title: qsTr("Import a community")
+
+
+    Input {
+        id: keyInput
+        label: qsTr("Community key")
+        placeholderText: qsTr("0x...")
+        validationError: popup.keyValidationError
+    }
+
+    footer: StatusButton {
+        text: qsTr("Import")
+        anchors.right: parent.right
+        onClicked: {
+            if (!validate()) {
+                return
+            }
+
+            let communityKey = keyInput.text
+            if (!communityKey.startsWith("0x")) {
+                communityKey = "0x" + communityKey
+            }
+
+            const error = chatsModel.importCommunity(communityKey)
+
+            if (error) {
+                creatingError.text = error
+                return creatingError.open()
+            }
+
+//            popup.close()
```

Comment:
Is this comment intentional?

Reply:
Ah, yes and no. It's because the import does nothing for now because status-go is/was messed up.
I'll remove it